types/ember-feature-flags/index.d.ts

```diff
@@ -14,4 +14,5 @@ export default interface Features extends Service {
     enable(feature: string): void;
     disable(feature: string): void;
     isEnabled(feature: string): boolean;
+    flags: string[];
```

Comment:
This can be marked `readonly`, since the `flags` property doesn't allow setting:

```suggestion
    readonly flags: string[];
```

Reply:
Thanks!